ISISScientificComputing/autoreduce
# django orm outside of webapp

### Summary of work
* Add functionality to access Django ORM
* Add `model` package we should also refactor `Message` into here (#540)
* Add test cases 


### How to test your work
* Ensure test cases and travis run
* You could try and write a small script to check this is working:
```
from model.database import DjangoORM
orm = DjangoORM()
orm.connect()
gem = orm.data_model.Instrument.objects.filter(name='GEM').first()
print(gem)
```

### Additional comments

Fixes #535
